src/Raven.Client/Documents/Session/ClusterTransactionSession.cs

```diff
@@ -97,7 +99,7 @@ protected async Task<CompareExchangeValue<T>> GetCompareExchangeValueAsyncIntern
             {
                 _session.IncrementRequestCount();
 
-                var value = await _session.Operations.SendAsync(new GetCompareExchangeValueOperation<BlittableJsonReaderObject>(key, materializeMetadata: false), sessionInfo: _session._sessionInfo, token: token).ConfigureAwait(false);
+                var value = await _session.Operations.SendAsync(new GetCompareExchangeValueOperation<BlittableJsonReaderObject>(key, materializeMetadata: true), sessionInfo: _session._sessionInfo, token: token).ConfigureAwait(false);
```

Comment:
can you explain why this change is needed?

Reply:
Yes,
From what I see now it does not have any influence on the tests,
The command response has metadata which is represented as a 'MetadataAsDictionary'.
If the 'materializeMetadata' is false, the MetadataAsDictionary object gets blittable in its constructor, and every time you try to get a value from the 'MetadataAsDictionary', the 'MetadataAsDictionary' gives you the value from the blittable,
and if it's true, the 'MetadataAsDictionary' object has an actual dictionary and its kvp's initialized (copied) from the blittable.

Reply:
so we want to revert this specific change? @ppekrol